src/Subscriber/MismatchBranchDescriptionSubscriber.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Subscriber;
+
+use App\Api\Issue\IssueApi;
+use App\Event\GitHubEvent;
+use App\GitHubEvents;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\String\UnicodeString;
+
+/**
+ * @author Antoine Makdessi <amakdessi@me.com>
```

Comment:
I will be your coauthor with great pleasure but please add your email also as taking care on continuation!